### PR TITLE
Update IDEIntegration.md

### PR DESCRIPTION
**Pull request by akamanocha**
As asked for PR for verified IDE support-

Tested batect Sample Java Project with the VS Code and other demo Spring boot Project (with the batect sample java project, tooling is bit slow but nonethless works)

Also it is language agnostic and hence should provide support for node, Python et al.

### CODE REVIEW

**Comment by charleskorn** [comment]
Hey @akamanocha, thanks for the PR and sorry for the slow response. I haven't had time to review this yet but I'll take a look over the weekend.

File: docs/content/tips/IDEIntegration.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Some solutions for this include:
   are known to work, although notably IntelliJ does not currently support this). There's more information on how to configure
   this in the [PyCharm docs](https://www.jetbrains.com/help/pycharm/configuring-remote-interpreters-via-docker.html) and
   [RubyMine docs](https://www.jetbrains.com/help/ruby/configuring-remote-interpreters-via-docker.html).
+* The latest release branch of Visual Studio Code - code named [Insiders](https://code.visualstudio.com/insiders/) and also     1.33+ has got this new extension - [Visual Studio Code Remote - Containers extension](https://code.visualstudio.com/docs/remote/containers#_creating-a-devcontainerjson-file) and with the same it gives the option   to use local filesystem and code in runnig docker container with runtme and tools. All the extensions and the IDE features,   including full IntelliSense (completions), code navigation, and debugging  can be used.
```
Comment (charleskorn) [inline docs/content/tips/IDEIntegration.md]: Could you please fix the typos `runnig` and `runtme`?

Comment (charleskorn) [reply docs/content/tips/IDEIntegration.md]: Can `The latest release branch of Visual Studio Code - code named [Insiders](https://code.visualstudio.com/insiders/) and also     1.33+ has got this new extension...` be rephrased to be `Visual Studio Code 1.33 or later, when running with the extension...`? Is that still valid? Or is it only available on the Insiders branch?

Comment (charleskorn) [reply docs/content/tips/IDEIntegration.md]: Please capitalise Docker (rather than `docker`).

**Comment by charleskorn** [comment]
I've made a few minor changes for clarity - once the build passes, I'll merge this PR.

Thanks again for your feedback and contribution!

**Comment by akamanocha** [comment]
> I've made a few minor changes for clarity - once the build passes, I'll merge this PR.
> 
> Thanks again for your feedback and contribution!

Sure thanks. 
I am bad with phrasing lines in English for simple tech documentation.
Surely, yours look better.

**Comment by charleskorn** [comment]
Not at all, I'm just very picky :)

**Pull request merged**